containers/production-schedule.yaml

```diff
@@ -0,0 +1,31 @@
+---
+# Run the mismatch finder scheduler on kubernetes
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
Is there a particular reason de decided to go with a deployment rather than a [`kind: CronJob`](https://wikitech.wikimedia.org/wiki/Help:Toolforge/Kubernetes#Kubernetes_cronjobs) pod here? I'm actually fine with both, but I wonder if there is a tradeoff between the two approaches.

Reply:
no you are right: I didn't pay attention and actually agree a cronjob that runs every minute is better than the continuous `schedule:work` job.